packages/wrangler/src/reporting.ts

```diff
@@ -1,7 +1,7 @@
-import { appendFile, readFile } from "fs/promises";
 import * as fs from "node:fs";
+import { readFile, writeFile } from "node:fs/promises";
 import os from "node:os";
-import path from "path/posix";
+import path from "node:path/posix";
```

Comment:
We should probably drop the `posix` here.

Reply:
done